firedrake/slope_limiter/limiter.py

```diff
@@ -0,0 +1,121 @@
+from firedrake import *
```

Comment:
Start with `from __future__ import absolute_import`

Please don't import everything inside internal modules, just grab the bits you need.